fabric/src/main/java/me/lunaluna/fabric/elytrarecast/ElytraHelper.java

```diff
@@ -20,18 +19,18 @@ public static boolean castElytra(ClientPlayerEntity player) {
     }
 
     public static boolean checkElytra(ClientPlayerEntity player) {
-        if (player.input.jumping && !player.getAbilities().flying && !player.hasVehicle() && !player.isClimbing()) {
+        if (player.input.playerInput.jump() && !player.getAbilities().flying && !player.hasVehicle() && !player.isClimbing()) {
             var itemStack = player.getEquippedStack(EquipmentSlot.CHEST);
-            return itemStack.isOf(Items.ELYTRA) && ElytraItem.isUsable(itemStack);
+            return itemStack.isOf(Items.ELYTRA);
         } else
             return false;
     }
 
     public static boolean checkFallFlyingIgnoreGround(ClientPlayerEntity player) {
         if (!player.isTouchingWater() && !player.hasStatusEffect(StatusEffects.LEVITATION)) {
             var itemStack = player.getEquippedStack(EquipmentSlot.CHEST);
-            if (itemStack.isOf(Items.ELYTRA) && ElytraItem.isUsable(itemStack)) {
```

Comment:
`ElytraItem` is gone I didn't try hard to find a replacement for the `ElytraItem.isUsable` call removing it seemed fine enough.

Reply:
I'm going to merge this, though will add `LivingEntity.canGlideWith(itemStack, EquipmentSlot.CHEST)` after, ensuring the elytra is usable is useful for stopping packets being sent trying to glide when elytra breaks.